sdk/inc/azure/core/az_credentials.h

```diff
@@ -104,27 +104,54 @@ typedef struct
     az_span client_id;
     az_span client_secret;
     az_span scopes;
+    az_span authority;
   } _internal;
 } az_credential_client_secret;
 
 /**
- * @brief az_credential_client_secret_init initializes an az_credential_client_secret instance
- * with the specified tenant ID, client ID and client secret.
+ * @brief Initializes an az_credential_client_secret instance with the specified tenant ID, client
+ * ID and client secret.
+ *
+ * @param out_credential Reference to an #az_credential_client_secret instance to initialize,
+ * @param tenant_id An Azure tenant ID.
+ * @param client_id An Azure client ID.
+ * @param client_secret An Azure client secret.
  *
- * @param out_credential reference to a az_credential_client_secret instance to initialize
- * @param tenant_id an Azure tenant ID
- * @param client_id an Azure client ID
- * @param client_secret an Azure client secret
  * @return An #az_result value indicating the result of the operation:
- *         - #AZ_OK if successful
- *         - Other error code if initialization failed
+ *         - #AZ_OK if successful.
+ *         - Other error code if initialization failed.
```

Comment:
nit: since these are sentence fragments in lists, they don't need to end in a period. At least that's what I have seen and been following across all the other API docs.
```suggestion
 *         - #AZ_OK if successful
 *         - Other error code if initialization failed
```